### PR TITLE
skill calc: Add common chests

### LINKED ISSUE #9938: Add thievable chests to Thieving xp calculator
The wiki has a relevant list:  https://oldschool.runescape.wiki/w/Thieving#Thievable_chests

### PR DESCRIPTION
**Pull request by dekvall**
Add the following chests:
 - Nature rune chest
 - Lizardman stone chest
 - Dorgesh-kaan rich chest
 - Rouges' Castle chest

I don't believe other chest are really used.
Closes #9938

### CODE REVIEW

File: runelite-client/src/main/resources/net/runelite/client/plugins/skillcalculator/skill_thieving.json [hunk 1]
```diff
@@ -210,12 +222,24 @@
       "name": "Gems Stall",
       "xp": 160
     },
+    {
+      "level": 78,
+      "icon": 1623,
+      "name": "Dorgesh-kaan rich chest",
```
Comment (Zirro) [inline runelite-client/src/main/resources/net/runelite/client/plugins/skillcalculator/skill_thieving.json]:
```suggestion
      "name": "Dorgesh-Kaan Rich chest",
```

File: runelite-client/src/main/resources/net/runelite/client/plugins/skillcalculator/skill_thieving.json [hunk 2]
```diff
@@ -210,12 +222,24 @@
...
     {
       "level": 80,
       "icon": 3259,
       "name": "Hero",
       "xp": 275
     },
+    {
+      "level": 84,
+      "icon": 1615,
+      "name": "Rouges' Castle chest",
```
Comment (Zirro) [inline runelite-client/src/main/resources/net/runelite/client/plugins/skillcalculator/skill_thieving.json]:
```suggestion
      "name": "Rogues' Castle chest",
```

File: runelite-client/src/main/resources/net/runelite/client/plugins/skillcalculator/skill_thieving.json (hunk 1 above)
Comment (oplosthee) [reply runelite-client/src/main/resources/net/runelite/client/plugins/skillcalculator/skill_thieving.json]: Might as well capitalize the `c` in `chest` while you're at it to stay consistent with the rest of the file

**Comment by dekvall** [comment]
Capitalized words and fixed typos, thanks guys!

**Comment by jachymb** [comment]
Note: The Hemenster chest is used for Medium kandarin diary.

**Comment by dekvall** [comment]
Is it something people commonly use to train? Since this will only affect the skill calculator.

**Comment by jachymb** [comment]
IDK, I used it to get low lvl fletching supplies on ironman it somewhat made sense.

**Comment by dekvall** [comment]
I feel like we don't need to clutter the UI more than necessary and I have left out other chests as well.
Neither the gp or xp of the Hemenester chest is great, I think it will be fine without it.

**Comment by Nightfirecat** [comment]
Added in 1294984dcefca9511d9e278c1bce2161a212c601

**Pull request closed without merging**